pkg/virt-launcher/notify-client/client.go

```diff
@@ -152,6 +155,17 @@ func StartNotifier(virtShareDir string, domainConn cli.Connection, deleteNotific
 
 	domainConn.SetReconnectChan(reconnectChan)
 
+	client, err := NewDomainEventClient(virtShareDir)
+	if err != nil {
+		log.Log.Error("Unable to create domain event notify client")
+	}
+
+	qemuAgentCallback := func(domainName string, interfaceStatuses *[]api.InterfaceStatus) {
+		eventCallback(domainConn, util.NewDomainFromName(domainName, vmiUID), LibvirtEvent{}, client, deleteNotificationSent, interfaceStatuses)
+	}
+	// TODO: Should the polling interval be configurable? Where is a good place for this?
+	agentPoller := virtwrap.CreatePoller(domainConn, vmiUID, 10, qemuAgentCallback)
```

Comment:
should be configurable. 10 is quite aggressive

Reply:
using: 	qemuAgentPollerInterval := pflag.Duration("qemu-agent-poller-interval", 60, "Interval between consecutive qemu agent calls")

Done